bin/pdfarranger

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
```

Comment:
I [and Fedora](https://docs.fedoraproject.org/en-US/packaging-guidelines/#_shebang_lines) would prefer `#!/usr/bin/python3`

Reply:
Done and squashed into 48b5df6100f46ca641694e71fe4f8fa7d298edb4